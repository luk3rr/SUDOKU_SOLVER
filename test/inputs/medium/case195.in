000000003 000123500 900008040 620010000 000000090 803060050 004000100 380600000 070300000 
