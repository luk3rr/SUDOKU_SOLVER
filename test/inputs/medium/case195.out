Solving the following grid:
0 0 0 | 0 0 0 | 0 0 3 
0 0 0 | 1 2 3 | 5 0 0 
9 0 0 | 0 0 8 | 0 4 0 
------+-------+------
6 2 0 | 0 1 0 | 0 0 0 
0 0 0 | 0 0 0 | 0 9 0 
8 0 3 | 0 6 0 | 0 5 0 
------+-------+------
0 0 4 | 0 0 0 | 1 0 0 
3 8 0 | 6 0 0 | 0 0 0 
0 7 0 | 3 0 0 | 0 0 0 


Solution found :')

2 5 8 | 9 4 6 | 7 1 3 
7 4 6 | 1 2 3 | 5 8 9 
9 3 1 | 5 7 8 | 6 4 2 
------+-------+------
6 2 5 | 8 1 9 | 3 7 4 
4 1 7 | 2 3 5 | 8 9 6 
8 9 3 | 4 6 7 | 2 5 1 
------+-------+------
5 6 4 | 7 9 2 | 1 3 8 
3 8 9 | 6 5 1 | 4 2 7 
1 7 2 | 3 8 4 | 9 6 5 

Algorithm: BFS
Total time: 452 ms
Total expanded states: 64640
